thirdParty/cmake-modules/FindmallocMC.cmake

```diff
@@ -22,6 +22,8 @@
 #   mallocMC_INCLUDE_DIRS    - Include directories for the mallocMC headers.
 #   mallocMC_FOUND           - TRUE if FindmallocMC found a working install
 #   mallocMC_VERSION         - Version in format Major.Minor.Patch
+# and for individual COMPONENTS:
+#   mallocMC_<COMPONENT>_FOUND  - TRUE if FindmallocMC found that component
```

Comment:
original case as the [recommendation](http://www.cmake.org/cmake/help/v3.0/module/FindPackageHandleStandardArgs.html), good point. will also add this upstream (see link above).